cloud/disk_manager/internal/pkg/dataplane/transfer_tests/transfer_test.go

```diff
@@ -935,3 +936,62 @@ func TestTransferFromDiskToIncrementalSnapshotWhenBaseSnapshotIsSmall(t *testing
 	_ = transfer(t, ctx, from, to, true)
 	to.checkChunks(t, ctx, expectedChunks)
 }
+
+func TestTransferFromDiskToDiskWithFewChangedBlocksOnBigDisk(t *testing.T) {
```

Comment:
Lets have 'with random failure` test, and check that test is hanging without your fix

Reply:
We have random failures in this test, and the test indeed hangs without the fix.